media/static-bundles.json

```diff
@@ -1065,6 +1071,16 @@
       ],
       "name": "firefox_whatsnew_65_newsletter"
     },
+    {
+      "files": [
+        "js/base/uitour-lib.js",
+        "js/base/mozilla-fxa.js",
+        "js/base/mozilla-fxa-init.js",
```

Comment:
I think "js/base/mozilla-fxa-form.js" should also be included here?

The hidden "flow_id" and "flow_begin_time" field is currently left empty for everyone & the the form doesn't work as expected in Fx China repack.

Reply:
Great catch @l-hedgehog, I'll open a new issue for this.

Reply:
Filed https://github.com/mozilla/bedrock/issues/6945

Reply:
I wish I noticed this before the 66 release.

Our QA also just told me the "Congrats! You’re using the latest version of Firefox." message is not localized, looks like it is not copied over from previous lang files into whatsnew_66.lang.

Reply:
> Our QA also just told me the "Congrats! You’re using the latest version of Firefox." message is not localized, looks like it is not copied over from previous lang files into whatsnew_66.lang.

I'll file another issue for this, thanks @l-hedgehog

Reply:
@alexgibson Thank you for the very quick response!

Reply:
I've filed a separate issue for the untranslated string and CC'd folks https://github.com/mozilla/bedrock/issues/6947